.drone.star

```diff
@@ -78,7 +78,7 @@ config = {
                 "oC10Files1": [
                     "webUIFilesCopy",
                     "webUIFavorites",
-                    "webUIMarkdownEditor",
+                    "webUISimpleEditor",
```

Comment:
@diocas that would need a rename of https://github.com/owncloud/web/tree/master/tests/acceptance/features/webUIMarkdownEditor in order to work :) since it's mapping feature files/folders